main.cpp

```diff
@@ -57,50 +89,55 @@ struct MidiKeyboard
     bool usbCompatable = true;
     bool metalBody = false;
     
-    void playMidiNotes(int keys);
-    void changePitch(bool pitchWheel);
-    void controlMacros(bool modWheel);
+    void playMidiNotes(bool keyPressed);
+    void changePitch(bool pitchWheelUp);
+    void controlMacros(bool modWheelUp);
+
 };
+void MidiKeyboard::playMidiNotes(bool)
+{
+  
+}
+void MidiKeyboard::changePitch(bool)
+{
+
+}
+void MidiKeyboard::controlMacros(bool)
+{
+
+}
 /*
- 2)guitar
- 5 properties:
- 1)manufacturer
- 2)neck
- 3)strings
- 4)jack
- 5)cable
- 3 things it can do:
- 1)play notes
- 2)play chords
- 3)play muted
  */
 # include <string>
+
 struct Guitar
 {
     std::string manufacturer = "akai";
     bool neck = true;
-    int strings = 6;
+    bool steel = true;
```

Comment:
Go back to `int strings = 6;`.  you have too many boolean variables for your 5 properties.
- [ ] revert to previous member variable.